tools/test/travis-ci/scancode-evaluate.py

```diff
@@ -112,7 +126,8 @@ def license_check(scancode_output_path):
             # check the next file in the scancode output
             continue
 
-        if not has_permissive_text_in_scancode_output(scancode_output_data_file['licenses']):
+        if not (has_permissive_text_in_scancode_output(scancode_output_data_file['licenses'])
+                or has_binary_license(scancode_output_data_file)):
```

Comment:
I think we use `black` for checking code formatting. It will suggest the much more readable formatting below:
```suggestion
        if not (
            has_permissive_text_in_scancode_output(
                scancode_output_data_file["licenses"]
            )
            or has_binary_license(scancode_output_data_file)
        ):
```